src/ServerConfigParams.ts

```diff
@@ -61,7 +61,7 @@ export interface ServerConfigParams {
      * @option false - The smart wallet of the relay user will be charged for the transaction
      * @option true - The relay worker will pay transaction gas.
      */
-    disableSponsoredTx: boolean;
+    disableSponsoredTx?: boolean;
```

Comment:
I'm not sure we need to make this optional. I think we could just set a default value if they're not specified, the same way as we do for `feesReceiver`.

Reply:
updated